docs/study-def-dates.md

```diff
@@ -59,7 +60,25 @@ The available options (hopefully self-explanatory) are:
 
 Note that NHS financial (reporting) year runs from 1st April to 31st March.
 
-Intervals of time can be added or subtracted from the index date (or from a function applied to the index date).
+Any `index_date` you've defined in your study definition can be overridden in your `project.yaml`, by providing an `--index-date-range` argument, like this:
+
+```yaml
+actions:
+
+  generate_study_population_1:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition --index-date-range "2020-01-01"
```

Comment:
How about...

```suggestion
    run: >
      cohortextractor:latest generate_cohort
        --study-definition study_definition
        --index-date-range "2020-01-01"
```

I didn't know it was possible to pass a single date to `--index-date-range`!